### ISSUE TITLE
Empty response on erros or if method is not supported

### ISSUE THREAD

**Issue by SpontaneousOverthrow** [emeraldpay/dshackle#147]
For example:
```
curl --data '{"jsonrpc":"2.0", "method": "eth_unexpected","params":[],"id":1}' -H "Content-Type: application/json" -X POST http://127.0.0.1:8545/
```

Infura's response: 
```
{
    "jsonrpc": "2.0",
    "id": 1,
    "error": {
        "code": -32601,
        "message": "The method eth_unexpected does not exist/is not available"
    }
}
```
---

```
curl --data  {"jsonrpc":"2.0","method":"eth_getLogs","params":[{"topics":["unexpected"]}],"id":1} -H "Content-Type: application/json" -X POST http://127.0.0.1:8545/
{
    "jsonrpc": "2.0",
    "id": 1,
    "result": []
}
```

Infura's response:
```
{
    "jsonrpc": "2.0",
    "id": 1,
    "error": {
        "code": -32600,
        "message": "data types must start with 0x"
    }
}
```

**Comment by splix** [comment]
I think I was fixing that, but maybe it's still on `master` and not included in the latest release. But I need to check. Which version do you use?

**Comment by SpontaneousOverthrow** [comment]
> I think I was fixing that, but maybe it's still on `master` and not included in the latest release. But I need to check. Which version do you use?

`dshackle:0.11.1`

**Comment by SpontaneousOverthrow** [comment]
> yeah, i see it's not in the release. fixed in [[2a3d7ec](https://github.com/emeraldpay/dshackle/commit/2a3d7ecb244c0f1111cb77c2c43e4d7eb74863f4)], but only in master yet

gonna check it now

**Comment by SpontaneousOverthrow** [comment]
It works great thanks!
